netstandard/pkg/NETStandard.Library.dependencies.props

```diff
@@ -144,7 +144,7 @@
       <TargetFramework>netcoreapp2.0</TargetFramework>
     </_dependency>
     <_dependency Include="$(PlatformPackageId)">
-      <TargetFramework>uap10.1</TargetFramework>
+      <TargetFramework>uap10.0.15138</TargetFramework>
```

Comment:
For example, in dir.props set `<UapRs3Tfm>uap10.0.15138</UapRs3Tfm>` an use that everywhere where we need this string.